packages/web/src/engine/globals/path.js

```diff
@@ -10,7 +10,7 @@ module.exports = function (app, engine) {
             req = req || this.lookup('request');
             return engine.env === 'server'
                 ? path
-                : utils.relUrlPath(path, _.get(req, 'path', '/'), app.web.get('builder.urls'));
+                : `./${ utils.relUrlPath(path, _.get(req, 'path', '/'), app.web.get('builder.urls')) }`;
```

Comment:
what happens if the path returned from `utils.relUrlPath` is already relative? e.g. does `./../../some/path` behave the same as `../../some/path`?

Reply:
There is a [test](https://github.com/frctl/fractal/pull/1062/commits/12b63f69abdcf64918304a99464f723fcca01570#diff-4b766753284eb822a5a32a1b8e1f55480b64bdcae906a926dfc0b5cbfff955d1R322) for that case, it passes